File: php/SivitasAkademik.php
<?php

class SivitasAkademik extends Human
{
    private $asal_universitas;
    private $email_edu;

    function __construct($asal_universitas = null, $email_edu = null)
    {
        $this->asal_universitas = $asal_universitas;
        $this->email_edu = $email_edu;
    }

    function setAsal($asal_universitas)
    {
        $this->asal_universitas = $asal_universitas;
    }

    function getAsal()
    {
        return $this->asal_universitas;
    }

    function setEmail($email_edu)
    {
        $this->email_edu = $email_edu;
    }

    function getEmail()
    {
        return $this->email_edu;
    }
}

?>

File: python/sivitasakademik.py
from human import human


class sivitasakademik(human):
    __asaluniversitas = ""
    __emailedu = ""

    def __init__(self, asaluniversitas="", emailedu=""):
        self.__asaluniversitas = asaluniversitas
        self.__emailedu = emailedu

    def setAsal(self, asaluniversitas):
        self.__asaluniversitas = asaluniversitas

    def getAsal(self):
        return self.__asaluniversitas

    def setEmail(self, emailedu):
        self.__emailedu = emailedu

    def getEmail(self):
        return self.__emailedu
